_posts/2024-11-24-Bulk.markdown

```diff
@@ -27,3 +27,15 @@ Later in the week I set off on doing some site cleanup (added an icon for the br
 
 -Git gets easier. That sounded funny, but in reality it isn't just git. Everything gets easier with consistency. It doesn't take long to get the hang of these things. A very refreshing prospect.
 
+## End of Week 3 12/6/24
+Another week down. Time flies, huh? Well, we finished up our first iteration of the "Line Follower." And just for future reference, when I say we I mean my dad and I. He is a software engineer turned teacher. Not really, but he's been the one showing me the ropes. I am getting better at grasping the concepts, using git, and coding in general. He has been a huge help in all of those things. Anyway, the rudementary "motorturn" code block was a fun challenge. We had a black line made of electrical tape, and a color sensing robot on treads. We wanted it to follow the line as accurately as possible. It was a fun combo of software "engineering" as my dad calls it (the process of writing concise and well organized code which is easy to read and manipulate) and using critical thinking skills to work around the big elephant in the room:we had one color sensor. As one with basic coding knowledge would surely understand, the presence of multiple color sensors would allow for an easy solution. Something along the lines of, "if they both detect black, bingo! If not, well, rotate until they both detect black!" Not that the real process was much more complex, but yours truly is very new to robotics. Thankfully, I was able to put together some functional code and get it rolling (in case you were curious the final code is in my repository "starterlego"). 
```

Comment:
nit ("nit" means "this is a small issue, but I'd do it slightly differently").

I'd change:

```
when I say we I mean my dad and I
```
to:
```
when I say "we", I mean "my dad and I"
```

Reply:
I'd make the following a link (as well as "linefollower", etc):

```
the rudementary "motorturn" code block
```

You can make links in hypertext by doing:

```
[linktext](linkurl)
```

Github makes it easy - just select the word you want to make into a link, and click "cmd-K".  So you'd have:

```
[motorturn](https://github.com/MarkMavis26/starterlego/blob/main/motorturn.py)
```

You might also want to cleanup/organize the scripts a bit - make sure they're named what you're calling them (motorturn, linefollower, etc), and link to them where referenced.  The linking will help keep you honest as far as clean code, good git practices, comments, etc.  ie, "other people will be reading this".

Reply:
nit - space after colon:

```
in the room:we had one
```